resume-buddy/src/main/webapp/scripts/script.js

```diff
@@ -13,19 +13,21 @@ function login() {
     fetch('/login').then(response => response.json()).then((login) => {
         const loginLinkElement = document.getElementById('login-container');
         const myAccountElement = document.getElementById('my-account');
+        const greetingElement = document.getElementById('greeting-container');
         if (login.isValidUser) {
             // Show the 'my account' option
             // Show a log out option
-            myAccountElement.style.display = "block";
-            loginLinkElement.innerHTML = "<a style=\"text-decoration:none\" href=\"" + 
-            login.logout_url + "\">Log Out</a>";
-            
+            greetingElement.innerHTML = "Welcome " + login.email + "!";
+            myAccountElement.innerHTML = "<a href=\"resume-review.html\">My Account</a>" +
+            "  •  " + "<a href=\"" + login.logout_url + "\">Log Out</a>";
+            loginLinkElement.style.display = "none";
         }
         else {
             // Show the log in option
-            loginLinkElement.innerHTML = "<a style=\"text-decoration:none\" href=\"" + 
+            loginLinkElement.innerHTML = "<a href=\"" + 
```

Comment:
Same thing here, `#login-container` is commented out so `loginLinkElement` is `null`.

Reply:
Yup sorry I moved these elements in the HTML and forgot to delete the commented out version, I'll do that now!